esf_main.tex

```diff
@@ -13,55 +13,28 @@
 \subsection{Ready-To-Drive-Sound (RTDS)}
 \subsubsection{Description}
 %Describe your concept of the RTDS, how is the sound produced, what are the parameters for activating the RTDS, etc.
-The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://cirlw3yrzg4afc1x47lv2k18.wpengine.netdna-cdn.com/wp-content/uploads/2016/12/WT-1205-new.pdf}{Link: Soberton Inc. WT-1205}). The buzzer automatically makes a noise when given power, with the loudness proportional
-to the voltage. The last step in the start up sequence will notify the CAN system it is time for the ready
-to drive sound. Then the corresponding node on the buzzer will close a relay between TS+, after a 1.2 K$\Omega$
-resistor, and the buzzer for two seconds. The resistor limits the voltage over the buzzer to 48V
-and the current to 20 mA.
+The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.puiaudio.com/pdf/XL-5530L-LW300-S-R.pdf}{Link: PUI Audio Inc. XL-5530L-LW300-S-R}). The buzzer automatically makes a noise when given power, with the loudness proportional
+to the voltage. The last step in the start up sequence will notify the CAN system that it is time for the ready
```

Comment:
This should say that the BMS will notify the Dashboard CAN node when the shutdown circuit is complete.